src/App.tsx

```diff
@@ -3,29 +3,92 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getToods } from './api/api';
 
 interface State {
   selectedUserId: number;
+  todos: Todo[];
 }
 
 class App extends React.Component<{}, State> {
   state: State = {
     selectedUserId: 0,
+    todos: [],
+  };
+
+  componentDidMount() {
+    this.loader();
+  }
+
+  loader = async () => {
+    this.setState({ todos: await getToods });
+  };
+
+  onSelectedUserId = (selectedUserId: number) => {
+    this.setState({ selectedUserId });
+  };
+
+  sortByTitle = async (title: string) => {
+    await this.loader();
+
+    this.setState(state => (
+      { todos: state.todos.filter((todo: Todo) => todo.title.includes(title)) }
+    ));
+  };
+
+  sortByCompleted = async (by: string) => {
```

Comment:
when sorting by the title it should sort only within sorted by status todos
![image](https://user-images.githubusercontent.com/57598825/151188790-d011953b-be16-46d3-9e06-a36fce667131.png)